From b3e865e02e9261d7cb5f655d5dee63558666f5b1 Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 15:59:34 +0100
Subject: [PORT FROM MAIN][BT] Multidevice support: BCM vendor specific library configuration

BZ: 76413

Configuring Broadcom vendor specific library to be built along with
bluedroid, necessary to support Broadcom BT chipset

Change-Id: I4e4c62e6e93a8af9ed7f9a47492bbd7b8986aabb
Orig-Change-Id: Ic3158560c18e49eb4c69b2d9027ce2c30722ee41
Category: device enablement
Domain: CWS.BT-Common
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-on: http://android.intel.com:8080/123798
Reviewed-by: Poignet, Cedric <cedric.poignet@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
---
 Android.mk |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/Android.mk b/Android.mk
index 8c8f711..c7fe54d 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,7 +1,5 @@
 LOCAL_PATH := $(call my-dir)
 
-ifeq ($(BOARD_HAVE_BLUETOOTH_BCM), true)
-
 include $(CLEAR_VARS)
 
 BDROID_DIR := $(TOP_DIR)external/bluetooth/bluedroid
@@ -43,5 +41,3 @@ endif
 ifeq ($(TARGET_PRODUCT), full_wingray)
     include $(LOCAL_PATH)/conf/moto/wingray/Android.mk
 endif
-
-endif # BOARD_HAVE_BLUETOOTH_BCM
-- 
1.7.4.1

